src/realm-settings/NewAttributeSettings.tsx

```diff
@@ -114,7 +114,7 @@ export default function NewAttributeSettings() {
     }));
 
   useFetch(
-    () => adminClient.users.getProfile({ realm }),
+    adminClient.users.getProfile,
```

Comment:
Not a fan of this as it makes the contract between `useFetch` and it's callback implicit. This means that if the callback is changed to accept some parameters then they will automatically be passed, leading to possible undefined behavior.

```suggestion
    () => adminClient.users.getProfile(),
```